tuned/admin/admin.py

```diff
@@ -354,24 +355,36 @@ def _action_recommend_profile(self):
 		print(self._profile_recommender.recommend())
 		return True
 
-	def _action_dbus_verify_profile(self, ignore_missing):
+	def _action_dbus_verify_profile(self, ignore_missing, verbose=False):
+		profile_name = self._get_active_profile()
+		if (not profile_name):
```

Comment:
This introduces regression in the following situation:
- no active profile set
- post-load profile set to some profile

Reproducer:
```
# systemctl stop tuned
# echo balanced > /etc/tuned/post_loaded_profile
# > /etc/tuned/active_profile
# echo manual > /etc/tuned/profile_mode
# systemctl start tuned
# tuned-adm verify
```
Note: test coverage for such scenario would be handy, but it's not related to this PR
```